ui/tests/acceptance/client-dashboard-test.js

```diff
@@ -11,23 +12,21 @@ import ss from 'vault/tests/pages/components/search-select';
 import { clickTrigger } from 'ember-power-select/test-support/helpers';
 import { ARRAY_OF_MONTHS } from 'core/utils/date-formatters';
 import { formatNumber } from 'core/helpers/format-number';
+import { staticNow } from 'vault/tests/helpers/stubs';
 
 const searchSelect = create(ss);
 
-const CURRENT_DATE = new Date();
-const LAST_MONTH = startOfMonth(subMonths(CURRENT_DATE, 1));
-const COUNTS_START = subMonths(CURRENT_DATE, 12); // pretend vault user started cluster 1 year ago
-
 // for testing, we're in the middle of a license/billing period
-const LICENSE_START = startOfMonth(subMonths(CURRENT_DATE, 6));
+const LICENSE_START = startOfMonth(subMonths(staticNow, 6)); // 2022-07-01
 // upgrade happened 1 month after license start
-const UPGRADE_DATE = addMonths(LICENSE_START, 1);
+const UPGRADE_DATE = addMonths(LICENSE_START, 1); // 2022-08-01
 
 module('Acceptance | client counts dashboard tab', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
   hooks.before(function () {
+    sinon.stub(Date, 'now').returns(staticNow);
```

Comment:
Must stub in `hooks.before` not `hooks.beforeEach` or you get an error from sinon about wrapping a method which is already wrapped